### ISSUE TITLE
Double Check is not correct.

### ISSUE THREAD

**Issue by hakanaltindis** [keremozer/aspnetcore-google-recaptcha-v3#1]
Hi,

You wrote "double check" comment in Index.cshtml file. But if you check in it, you cannot re-validate in the controller file in server-side.

I think, you can refactor that.

**Comment by keremozer** [comment]
Thank you for feedback.

I removed the comment in the index.cshtml.

Thanks.

**Comment by hakanaltindis** [comment]
Hi again,

I told myself wrong 

Not only comment. You should this code block. If you validate token in cshtml file, you will not re-validate in post action(cs file). 


`
$.getJSON("@Url.Page("Index","RecaptchaVerify")&token=" + token, function (data) {
  $('#newsletter-form').find("input[type=submit]").prop("disabled", !data);
});
`

**Comment by keremozer** [comment]
Hi,
I tried to make two validation (client and server-side). When the document/ReCaptcha(js) is ready, I check it on the client-side. If ReCaptcha is not valid on the client-side, I disabled the button (I don't want the form to submit the server if the reCaptcha not valid.). 
If the user enables ownself and submits the button, therefore, I also check the ReCaptcha on the server-side.

as far as understand I tried to explain 

Thank you.
